frontend/src/app/commonComponents/Checkboxes.tsx

```diff
@@ -33,20 +36,30 @@ const Checkboxes = (props: Props) => {
     legendSrOnly,
     validationStatus,
     errorMessage,
+    required,
   } = props;
   const checkboxIds = useUniqueIds("check", boxes.length);
 
   return (
     <fieldset
-      className={classnames("usa-fieldset prime-checkboxes", props.className)}
+      className={classnames(
+        "usa-fieldset prime-checkboxes",
+        validationStatus === "error" && "usa-form-group--error",
+        props.className
+      )}
     >
       <legend
-        className={classnames("usa-legend", legendSrOnly && "usa-sr-only")}
+        className={classnames(
+          "usa-legend",
+          validationStatus === "error" && "usa-label--error",
+          legendSrOnly && "usa-sr-only"
+        )}
       >
-        {legend}
+        {required ? <Required label={legend} /> : <Optional label={legend} />}
       </legend>
       {validationStatus === "error" && (
-        <div role="alert" className="usa-error-message">
+        <div className="usa-error-message" role="alert">
```

Comment:
⚠️ will make the error message into its own component to reuse for various form inputs